catalog/ui/search-ui/standard/src/main/webapp/js/model/user.js

```diff
@@ -94,14 +94,15 @@ define([
             };
         },
         parse: function(resp) {
-            resp.label = 'Type: ' + resp.type;
-            if (resp.layer) {
-                resp.label += ' Layer: ' + resp.layer;
+            var layer = _.clone(resp);
```

Comment:
Eventually the layer name will be use in other views.  It would be nice to keep that logic in the model instead of duplicating it in all views that need it.